content/yaml/building-a-native-ios-app.md

```diff
@@ -22,12 +22,28 @@ For building an unsigned iOS app (.app), you need to run the following command i
                        CODE_SIGNING_REQUIRED=NO \
                        CODE_SIGNING_ALLOWED=NO
 
+If you don't have a workspace, use `-project "MyXcodeProject.xcodeproj"` instead of the `-workspace "MyXcodeWorkspace.xcworkspace"` option.
+
+Your artifact will be generated at the default Xcode path (unless you explicitly specify another path using the `CONFIGURATION_BUILD_DIR` option) by adding the following pattern in the `artifacts` section of `codemagic.yaml`:
```

Comment:
> Your artifact will be generated at the default Xcode path /.../ by adding the following pattern in the artifacts section of ...

This doesn't make sense. It implies that specifying the pattern in artifact paths somehow makes Xcode to save the artifact there.

Reply:
fixed here 79cf5b8